java/ingest/src/test/java/sleeper/ingest/IngestRecordsFromIteratorTest.java

```diff
@@ -58,8 +65,11 @@ public void shouldWriteMultipleRecords() throws Exception {
                 Arrays.asList(rootPartition, partition1, partition2));
 
         // When
-        IngestProperties properties = defaultPropertiesBuilder(stateStore, schema).build();
-        long numWritten = new IngestRecordsFromIterator(properties, getRecords().iterator()).write().getNumberOfRecords();
+        TableProperties tableProperties = createTableProperties(instanceProperties, schema, "");
+        IngestCoordinatorFactory factory = defaultFactory(inputFolderName, new FixedStateStoreProvider(tableProperties, stateStore),
+                instanceProperties, new FixedTablePropertiesProvider(tableProperties));
```

Comment:
Could we have a version of defaultFactory that takes the StateStore and TableProperties directly using FixedStateStoreProvider and FixedTablePropertiesProvider?